app.go

```diff
@@ -122,6 +122,11 @@ func NewHeadscale(cfg *Config) (*Headscale, error) {
 	var dbString string
 	switch cfg.DBtype {
 	case Postgres:
+		if cfg.DBurl != "" {
```

Comment:
Any particular reason for this to only apply to Postgres? This seems like this could be a generic feature to override DBURL for any DBType